node/tests/GlideClusterClient.test.ts

```diff
@@ -1488,6 +1488,66 @@ describe("GlideClusterClient", () => {
                             client.close();
                         }
                     });
+                    it(
```

Comment:
nit: add blank line between previous test and this one for readability
```suggestion
                    });
                    
                    it(
```